pydcs_extensions/llhmodpack/llhmodpack.py

```diff
@@ -0,0 +1,139 @@
+# Requires French Pack mod :
```

Comment:
Is there really a requirement to have French Pack installed? If it is so, this should be stated somewhere. Extra points if you disable the checkbox and only let it be checked if frenchpack is also checked if there is a dependency between those mods

Reply:
No requirements, just a typo.